be/src/exec/vectorized/aggregate/aggregate_blocking_node.cpp

```diff
@@ -188,13 +188,15 @@ std::vector<std::shared_ptr<pipeline::OperatorFactory>> AggregateBlockingNode::_
     size_t degree_of_parallelism = down_cast<SourceOperatorFactory*>(ops_with_sink[0].get())->degree_of_parallelism();
 
     auto should_cache = context->should_interpolate_cache_operator(ops_with_sink[0], id());
-    auto operators_generator = [this, should_cache, &context](bool post_cache) {
+    bool could_local_shuffle = context->could_local_shuffle(ops_with_sink);
```

Comment:
when local shuffle is interpolated,  query cache can not work. so when should_cache is true, turn off could_local_shuffle

```cpp
 bool could_local_shuffle =!should_cache && context->could_local_shuffle(ops_with_sink);
```

Reply:
Ok, I'll update